app/pages/patientdata/patientdata.js

```diff
@@ -498,7 +506,7 @@ export let PatientData = React.createClass({
     });
   },
 
-  handleSwitchToDailyPrintView: function() {
+  handleSwitchToPrintView: function() {
```

Comment:
based on comments in the code you should rename this

Reply:
done